sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -374,13 +377,13 @@ abstract class FixedHeightScrollable extends Scrollable {
   double _height;
   void _handleSizeChanged(Size newSize) {
     setState(() {
-      _height = newSize.height;
+      _height = scrollDirection == ScrollDirection.vertical ? newSize.height : newSize.width;
       scrollBehavior.containerSize = _height;
     });
   }
 
   void _updateContentsHeight() {
```

Comment:
Interestingly, updating the name of this method exposed another bug:
```
  void _updateContentsHeight() {
    double contentsHeight = itemHeight * itemCount;
    if (padding != null)
        contentsHeight += padding.top + padding.bottom; // NO because scrollDirection

    scrollBehavior.contentsSize = contentsHeight;
  }
```